File: JavaScript/01.JavaScript-Basic-February-2021/05.conditionalStatementsAdvanced-Lab/08.cinemaTicket.js
function cinemaTicket(input) {
    let day = input[0];

    if (day == "Monday" || day == "Tuesday" || day == "Friday") {
        console.log("12");
    } else if (day == "Wednesday" || day == "Thursday") {
        console.log("14");
    } else if (day == "Saturday" || day == "Sunday") {
        console.log("16");
    }
}

cinemaTicket(["Monday"]);
cinemaTicket(["Friday"]);
cinemaTicket(["Sunday"]);

File: Go/01.Go-Basic-February-2022/05.conditionalStatementAdvanced-Lab/08.cinemaTicket.go
package main

import "fmt"

func main() {
	var dayOfWeek string
	fmt.Scanln(&dayOfWeek)

	switch dayOfWeek {
	case "Monday", "Tuesday", "Friday":
		fmt.Print(12)
	case "Wednesday", "Thursday":
		fmt.Print(14)
	case "Saturday", "Sunday":
		fmt.Print(16)
	}
}
